alerts/resource_alerts.libsonnet

```diff
@@ -75,7 +75,7 @@
             'for': '5m',
           },
           {
-            alert: 'KubeMemOvercommit',
+            alert: 'KubeMemQuotaOvercommit',
```

Comment:
Nit: Wdyt of just calling this `KubeMemoryQuotaOvercommit`, anything against that?

Reply:
SGTM, changed both.